setup.py

```diff
@@ -16,8 +16,28 @@
 
 setup(name='haas',
       version='0.2rc2',
+      author='https://github.com/CCI-MOC/haas/network/members',
+      author_email = 'haas-dev-list@bu.edu',
       url='https://github.com/CCI-MOC/haas',
+      description='Hardware-as-a-Service or HaaS is a bare-metal isolation service that automates \
+allocation and management of non-virtualized compute resources',
+      long_description='HaaS allows mutually untrusting and incompatible bare metal services \
```

Comment:
fixed following the what @zenhack  suggested from 
https://github.com/zenhack/git-wat/blob/master/setup.py